### PR TITLE
[Docs] Add 1_exist_data_model.md Chinese Version

### PR DESCRIPTION
**Pull request by DCNSW**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation
Add 1_exist_data_model.md Chinese version.

## Modification
Add 1_exist_data_model.md Chinese version.

## BC-breaking (Optional)
Does the modification introduce changes that break the back-compatibility of the downstream repos?
no.

## Use cases (Optional)
no.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs_zh-CN/1_exist_data_model.md [hunk 1]
```diff
@@ -1 +1,220 @@
...
+- `RESULT_FILE`: 输出结果（pickle 格式）的文件名，如果未指定，结果不会被保存。
+- `EVAL_METRICS`: 在结果上评测的项，不同的数据集有不同的合法值。具体来说，我们默认对不同的数据集都使用各自的官方度量方法进行评测，所以对 nuScenes、Lyft、ScanNet 和 SUNRGBD 这些数据集来说可以简单设置为 `mAP`；对 KITTI 数据集来说，如果我们只想评测 2D 检测效果，可以将度量方法设置为 `img_bbox`；对于 Waymo 数据集，我们提供了 KITTI 风格（不稳定）和 Waymo 官方风格这两种评测方法，分别对应 `kitti` 和 `waymo`，我们推荐使用默认的官方度量方法，它的性能稳定而且可以与其它算法公平比较。
+- `--show`: 如果被指定，检测结果会在静默模式下被保存，用于调试和可视化，但只在单块GPU测试的情况下生效，和 `--show-dir` 搭配使用。
+- `--show-dir`: 如果被指定，检测结果会被保存在指定文件夹下的 `***_points.obj` 和 `***_pred.obj` 文件中，用于调试和可视化，但只在单块GPU测试的情况下生效，对于这个选项，图形化界面在你的环境中不是必需的。
+
+示例：
+
+假定你已经把模型权重文件下载到 `checkpoints/` 文件夹下：
+
+1. 在 ScanNet 数据集上测试 votenet，保存模型，可视化预测结果
```
Comment (Wuziyi616) [inline docs_zh-CN/1_exist_data_model.md:35]: 能否将 `votenet` 都改成 `VoteNet`？后面有出现也可以改一下；麻烦英文文档也做相应的改动

File: docs_zh-CN/1_exist_data_model.md [hunk 2]
```diff
@@ -1 +1,220 @@
...
+
+3. 在 ScanNet 数据集上测试 votenet（不保存测试结果），计算 mAP
+
+   ```shell
+   python tools/test.py configs/votenet/votenet_8x8_scannet-3d-18class.py \
+       checkpoints/votenet_8x8_scannet-3d-18class_20200620_230238-2cea9c3a.pth \
+       --eval mAP
+   ```
+
+4. 使用8块显卡测试 SECONDs, 计算 mAP
```
Comment (Wuziyi616) [inline docs_zh-CN/1_exist_data_model.md:60]: `SECONDs` --> `SECOND`

File: docs_zh-CN/1_exist_data_model.md [hunk 3]
```diff
@@ -1 +1,220 @@
...
+
+   ```shell
+   ./tools/slurm_test.sh ${PARTITION} ${JOB_NAME} configs/second/hv_second_secfpn_6x8_80e_kitti-3d-3class.py \
+       checkpoints/hv_second_secfpn_6x8_80e_kitti-3d-3class_20200620_230238-9208083a.pth \
+       --out results.pkl --eval mAP
+   ```
+
+5. 使用8块显卡在 nuscenes 数据集上测试 PointPillars, 生成提交给官方评测服务器的 json 文件
```
Comment (Wuziyi616) [inline docs_zh-CN/1_exist_data_model.md:68]: `nuscenes` --> `nuScenes`，英文文档也请改过来

File: docs_zh-CN/1_exist_data_model.md [hunk 4]
```diff
@@ -1 +1,220 @@
...
+# 单块显卡测试
+python tools/test.py ${CONFIG_FILE} ${CHECKPOINT_FILE} [--out ${RESULT_FILE}] [--eval ${EVAL_METRICS}] [--show] [--show-dir ${SHOW_DIR}]
+
+# 多块显卡测试
+./tools/dist_test.sh ${CONFIG_FILE} ${CHECKPOINT_FILE} ${GPU_NUM} [--out ${RESULT_FILE}] [--eval ${EVAL_METRICS}]
+```
+
+可选参数：
+- `RESULT_FILE`: 输出结果（pickle 格式）的文件名，如果未指定，结果不会被保存。
+- `EVAL_METRICS`: 在结果上评测的项，不同的数据集有不同的合法值。具体来说，我们默认对不同的数据集都使用各自的官方度量方法进行评测，所以对 nuScenes、Lyft、ScanNet 和 SUNRGBD 这些数据集来说可以简单设置为 `mAP`；对 KITTI 数据集来说，如果我们只想评测 2D 检测效果，可以将度量方法设置为 `img_bbox`；对于 Waymo 数据集，我们提供了 KITTI 风格（不稳定）和 Waymo 官方风格这两种评测方法，分别对应 `kitti` 和 `waymo`，我们推荐使用默认的官方度量方法，它的性能稳定而且可以与其它算法公平比较。
```
Comment (Wuziyi616) [inline docs_zh-CN/1_exist_data_model.md:27]: 之前往 MMDet3D 加入分割任务后没有修改这里的文档，可以加一下`对分割数据集如 ScanNet、S3DIS，可以设置为 'mIoU'`，同时英文文档也可以加一下

File: docs_zh-CN/1_exist_data_model.md [hunk 5]
```diff
@@ -1 +1,220 @@
...
+1. 通过 `--options` 设置端口，这是更推荐的，因为它不改变原来的配置
+
+   ```shell
+   CUDA_VISIBLE_DEVICES=0,1,2,3 GPUS=4 ./tools/slurm_train.sh ${PARTITION} ${JOB_NAME} config1.py ${WORK_DIR} --options 'dist_params.port=29500'
+   CUDA_VISIBLE_DEVICES=4,5,6,7 GPUS=4 ./tools/slurm_train.sh ${PARTITION} ${JOB_NAME} config2.py ${WORK_DIR} --options 'dist_params.port=29501'
+   ```
+
+2. 修改配置文件（通常在配置文件的倒数第6行）来设置不同的通信端口
+
+   In `config1.py`,
```
Comment (Wuziyi616) [inline docs_zh-CN/1_exist_data_model.md:203]: 在 `config1.py` 中

File: docs_zh-CN/1_exist_data_model.md [hunk 6]
```diff
@@ -1 +1,220 @@
...
+
+   ```python
+   dist_params = dict(backend='nccl', port=29500)
+   ```
+
+   In `config2.py`,
```
Comment (Wuziyi616) [inline docs_zh-CN/1_exist_data_model.md:209]: 在 `config2.py` 中

File: docs_zh-CN/1_exist_data_model.md [hunk 7]
```diff
@@ -1 +1,220 @@
-# 1: 使用已有模型在标准数据集上进行推理
+# 1: 使用已有模型在标准数据集上进行推理和训练
+
+## 使用已有模型进行推理
+
+这里我们提供了评测 SUNRGBD、ScanNet、KITTI 等多个数据集的测试脚本
```
Comment (Wuziyi616) [inline docs_zh-CN/1_exist_data_model.md:5]: 少了句号

File: docs_zh-CN/1_exist_data_model.md [hunk 8]
```diff
@@ -1 +1,220 @@
...
+
+请参考[开始](https://mmdetection3d.readthedocs.io/en/latest/getting_started.html)下的验证/样例来获取更容易集成到其它项目和基本样例的高级接口
```
Comment (Wuziyi616) [inline docs_zh-CN/1_exist_data_model.md:7]: 同样句号

File: docs/1_exist_data_model.md [hunk 9]
```diff
@@ -24,23 +24,23 @@ python tools/test.py ${CONFIG_FILE} ${CHECKPOINT_FILE} [--out ${RESULT_FILE}] [-
...
+- `EVAL_METRICS`: Items to be evaluated on the results. Allowed values depend on the dataset. Typically we default to use official metrics for evaluation on different datasets, so it can be simply set to `mAP` as a placeholder for detection tasks, which applies to nuScenes, Lyft, ScanNet and SUNRGBD. For KITTI, if we only want to evaluate the 2D detection performance, we can simply set the metric to `img_bbox` (unstable, stay tuned). For Waymo, we provide both KITTI-style evaluation (unstable) and Waymo-style official protocol, corresponding to metric `kitti` and `waymo` respectively. We recommend to use the default official metric for stable performance and fair comparison with other methods. Similarly, the metric can be set to `mIoU` for segmentation tasks, which applies to S3DIS、ScanNet.
```
Comment (THU17cyz) [inline docs/1_exist_data_model.md:27]: 中文盾号、 
可以改成 and

File: docs_zh-CN/1_exist_data_model.md [hunk 10]
```diff
@@ -1 +1,220 @@
...
+8. 使用8块显卡在 waymo 数据集上测试 PointPillars, 使用 waymo 度量方法计算 mAP
+
+   ```shell
+   ./tools/slurm_test.sh ${PARTITION} ${JOB_NAME} configs/pointpillars/hv_pointpillars_secfpn_sbn-2x16_2x_waymo-3d-car.py \
+       checkpoints/hv_pointpillars_secfpn_sbn-2x16_2x_waymo-3d-car_latest.pth --out results/waymo-car/results_eval.pkl \
+       --eval waymo --eval-options 'pklfile_prefix=results/waymo-car/kitti_results' \
+       'submission_prefix=results/waymo-car/kitti_results'
+   ```
+
+   **注意**: 对于 waymo 数据集上的评估，请根据[说明](https://github.com/waymo-research/waymo-open-dataset/blob/master/docs/quick_start.md/)构建二值文件 `compute_detection_metrics_main` 来做度量计算，并把它放在 `mmdet3d/core/evaluation/waymo_utils/`。（在使用 bazel 构建  `compute_detection_metrics_main` 时，有时会出现 `'round' is not a member of 'std'` 的错误，我们只需要把那个文件中 `round` 前的 `std::` 去掉。）二值文件生成时需要在 `--eval-options` 中给定 `pklfile_prefix`。对于度量方法，`waymo` 是推荐的官方评估策略，目前 `kitti` 评估是依照 KITTI 而来的，每个难度的结果和 KITTI 的定义并不完全一致。目前大多数物体都被标记为0难度，会在未来修复。它的不稳定原因包括评估的计算大、转换后的数据缺乏遮挡和截断、难度的定义不同以及平均精度的计算方法不同。
```
Comment (THU17cyz) [inline docs_zh-CN/1_exist_data_model.md:110]: 二进制文件这个名称是不是更通用一点？

File: docs_zh-CN/1_exist_data_model.md [hunk 11]
```diff
@@ -1 +1,220 @@
...
+9. 使用8块显卡在 waymo 数据集上测试 PointPillars, 生成 bin 文件并提交到排行榜
+
+   ```shell
+   ./tools/slurm_test.sh ${PARTITION} ${JOB_NAME} configs/pointpillars/hv_pointpillars_secfpn_sbn-2x16_2x_waymo-3d-car.py \
+       checkpoints/hv_pointpillars_secfpn_sbn-2x16_2x_waymo-3d-car_latest.pth --out results/waymo-car/results_eval.pkl \
+       --format-only --eval-options 'pklfile_prefix=results/waymo-car/kitti_results' \
+       'submission_prefix=results/waymo-car/kitti_results'
+   ```
+
+   **注意**: 生成 bin 文件后，你可以简单地构建二值文件  `create_submission`，并根据[说明](https://github.com/waymo-research/waymo-open-dataset/blob/master/docs/quick_start.md/)创建提交的文件。要在验证服务器上评测验证数据集，你也可以用同样的方式生成提交的文件。
```
Comment (THU17cyz) [inline docs_zh-CN/1_exist_data_model.md:121]: 同上，二值->二进制

File: docs_zh-CN/1_exist_data_model.md [hunk 12]
```diff
@@ -1 +1,220 @@
...
+所有的输出（日志文件和模型权重文件）都会被保存到工作目录下，通过配置文件里的 `work_dir` 指定。
+
+默认我们每过一个周期都在验证数据集上评测模型，你可以通过在训练配置里添加间隔参数来改变评测的时间间隔：
+
+```python
+evaluation = dict(interval=12)  # 每12个周期评估一次模型
+```
+
+**重要**：配置文件中的默认学习率对应8块显卡，配置文件名里有具体的批量大小，比如'2x8'表示一共8块显卡，每块显卡2个样本。
+根据 [Linear Scaling Rule](https://arxiv.org/abs/1706.02677)，如果你用不同数量的显卡或每块显卡有不同数量的图像，你需要依批量大小按比例调整学习率，比如，4块显卡、每块显卡2幅图像的情况下学习率应设为0.01，16块显卡、每块显卡4幅图像的情况下学习率应设为0.08。然而，由于大多数模型使用 ADAM 而不是 SGD 进行优化，上述规则可能并不适用，用户需要自己调整学习率。
```
Comment (THU17cyz) [inline docs_zh-CN/1_exist_data_model.md:136]: 这个因果关系是不是可以明显一点？如果...学习率0.01，那么...学习率0.08

File: docs_zh-CN/1_exist_data_model.md [hunk 13]
```diff
@@ -1 +1,220 @@
...
+
+可选参数：
+
+- `--no-validate` (**不推荐**): 默认情况下，代码在训练阶段每 k（默认值是1，可以像[这里](https://github.com/open-mmlab/mmdetection3d/blob/master/configs/fcos3d/fcos3d_r101_caffe_fpn_gn-head_dcn_2x8_1x_nus-mono3d.py#L75)一样修改）个周期做一次评测，如果要取消评测，使用 `--no-validate`。
+- `--work-dir ${WORK_DIR}`: 覆盖配置文件中的指定工作目录。
+- `--resume-from ${CHECKPOINT_FILE}`: 从之前的模型权重文件中恢复。
+- `--options 'Key=value'`: 覆盖使用的配置中的一些设定。
+
+`resume-from` 和 `load-from` 的不同点:
+`resume-from` 加载模型权重和优化器状态，同时周期数也从特定的模型权重文件中继承，通常用于恢复偶然中断的训练过程。
```
Comment (THU17cyz) [inline docs_zh-CN/1_exist_data_model.md:160]: 加一个无序列表的markdown结构？

File: docs_zh-CN/1_exist_data_model.md [hunk 14]
```diff
@@ -133,7 +133,7 @@ evaluation = dict(interval=12)  # 每12个周期评估一次模型
 ```
 
 **重要**：配置文件中的默认学习率对应8块显卡，配置文件名里有具体的批量大小，比如'2x8'表示一共8块显卡，每块显卡2个样本。
-根据 [Linear Scaling Rule](https://arxiv.org/abs/1706.02677)，如果你用不同数量的显卡或每块显卡有不同数量的图像，你需要依批量大小按比例调整学习率，比如，4块显卡、每块显卡2幅图像的情况下学习率应设为0.01，16块显卡、每块显卡4幅图像的情况下学习率应设为0.08。然而，由于大多数模型使用 ADAM 而不是 SGD 进行优化，上述规则可能并不适用，用户需要自己调整学习率。
+根据 [Linear Scaling Rule](https://arxiv.org/abs/1706.02677)，如果你用不同数量的显卡或每块显卡有不同数量的图像，就需要依批量大小按比例调整学习率，比如，如果用4块显卡、每块显卡2幅图像，学习率应设为0.01，如果用16块显卡、每块显卡4幅图像，学习率应设为0.08。然而，由于大多数模型使用 ADAM 而不是 SGD 进行优化，上述规则可能并不适用，用户需要自己调整学习率。
```
Comment (THU17cyz) [inline docs_zh-CN/1_exist_data_model.md:136]: 我的意思是4卡2图0.01-->16卡4图0.08

**Review by THU17cyz — APPROVED** [review]

**Review by Wuziyi616 — APPROVED** [review]

File: docs_zh-CN/1_exist_data_model.md (hunk 14 above)
Comment (Wuziyi616) [reply docs_zh-CN/1_exist_data_model.md:136]: 我觉得业臻的意思是改成 `比如，如果用4块显卡、每块显卡2幅图像，学习率应设为0.01，那么用16块显卡、每块显卡4幅图像，学习率应设为0.08`

**Comment by Wuziyi616** [comment]
Great! I think this PR is ready to merge now @ZwwWayne

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**